app/component/CustomizeSearch.js

```diff
@@ -246,6 +323,19 @@ class CustomizeSearch extends React.Component {
       />
     </section>);
 
+  getAccessibilityOption = () => {
+    let accessibilityOption;
+    if (getCustomizedSettings().accessibilityOption
+    && !this.context.location.query.accessibilityOption) {
+      accessibilityOption = getCustomizedSettings().accessibilityOption;
+    } else if (this.context.location.query.accessibilityOption) {
```

Comment:
I think this goes wrong. Query param should again overrule custSettings, and  if query.accessibilityOption===0, it will be ignored.